hs_core/views/serializers.py

```diff
@@ -0,0 +1,75 @@
+__author__ = 'Pabitra'
+
+from hs_core import hydroshare
+from django.contrib.auth.models import Group, User
+from rest_framework import serializers
+from .utils import authorize, validate_json, validate_user_name, validate_group_name
+from hs_core.hydroshare import utils
+
+
+class ResourceUpdateRequestValidator(serializers.Serializer):
+    title = serializers.CharField(required=False)
+    metadata = serializers.CharField(validators=[validate_json], required=False)
+    # TODO: use a custom validator for user names and group names
+    user_choices = [user.username for user in User.objects.all()]
+    group_choices = [grp.name for grp in Group.objects.all()]
+    edit_users = serializers.MultipleChoiceField(choices=user_choices, required=False)
```

Comment:
Yes the usual thing to do in a form is to set it to `YourModel.objects.none()` in the class definition and then do `self.fields[fieldname].choices = YourModel.objects.filter(...)` at run-time. You can follow a similar pattern for the serializer, I suspect.